AEPEdgeBridge.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seems like a new file. Do we need to commit this? @kevinlind @emdobrin

Reply:
Usually xcshareddata settings can be committed as shared settings for the project, but we ignore xcuserdata settings. Is this required for any of the changes you made @cacheung, if not let's revert it